core/src/main/java/de/danoeh/antennapod/core/glide/FastBlurTransformation.java

```diff
@@ -285,4 +286,9 @@ private static Bitmap fastBlur(Bitmap bitmap, int radius) {
         bitmap.setPixels(pix, 0, w, 0, 0, w, h);
         return bitmap;
     }
+
+    @Override
+    public int hashCode() {
```

Comment:
Could you move the method up, next to the equals method, please?

Reply:
OK!
I've done this.